examples/one-2-one/src/controllers/account.controller.ts

```diff
@@ -0,0 +1,120 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/example-account-list
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Filter, repository} from '@loopback/repository';
+import {
+  del,
+  get,
+  getFilterSchemaFor,
+  param,
+  patch,
+  post,
+  put,
+  requestBody,
+} from '@loopback/rest';
+import {Account, Supplier} from '../models';
+import {AccountRepository} from '../repositories';
+
+export class AccountController {
```

Comment:
Do we want to expose accounts directly? Have you considered removing `AccountController` and exposing only `SupplierAccountController` + `SupplierController`? The less files we have in the example, the easier it is to read for our users and the less files we have to maintain ourselves.